nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/FTPTransfer.java

```diff
@@ -304,6 +304,12 @@ public void flush() throws IOException {
     }
 
     @Override
+    public void flush(final FlowFile flowFile) throws IOException {
+        final FTPClient client = getClient(flowFile);
+        client.completePendingCommand();
```

Comment:
Should we handle the case where `client.completePendingCommand()` returns false? Per the javadocs [1], it returns false if the command couldn't be completed. 

[1] https://commons.apache.org/proper/commons-net/apidocs/org/apache/commons/net/ftp/FTPClient.html#completePendingCommand()